### PR TITLE
Populate remaining footer items

### LINKED ISSUE #174: Populate base template footer
Right now the footer only contains the version number, but it's not clearly offset from the rest of the content. We should consider:

- [x] Use an `<hr />` to separate it from the rest of the content
- [x] Use CSS padding or something to distance it from the rest of the content
- [ ] Populate footer with e.g.:
    - [x] Link to source code repository (#238)
    - [ ] Link to Benefit Tool documentation
    - [ ] Support contact information -- link to GitHub Discussions? I think this is a valuable way to add a support forum to the app. 
    - [x] Copyright owner
    - [ ] Copyright date

### PR DESCRIPTION
**Pull request by Sherwin-14**
I have added all the unchecked items. please have a look at it ;)

Closes #174

----
📚 Documentation preview 📚: https://usaon-benefit-tool--246.org.readthedocs.build/en/246/

### CODE REVIEW

File: usaon_benefit_tool/templates/base.html [hunk 1]
```diff
@@ -52,11 +52,23 @@
   <footer>
     <hr />
     <div class="footer-items">
-      <p>&copy; USAON &amp; NSIDC</p>
+      <p>&copy; USAON &amp; NSIDC 2022</p>
```
Comment (mfisher87) [inline usaon_benefit_tool/templates/base.html:55]:
```suggestion
      <p>&copy; USAON &amp; NSIDC 2024</p>
```

File: usaon_benefit_tool/templates/base.html [hunk 2]
```diff
@@ -52,11 +52,23 @@
...
+      &nbsp;
+      <a href="{{doc_url}}">
+        {{ render_icon('filetype-doc' )}}
+      </a>
+      &nbsp;
+      &nbsp;
+      <a href="{{discuss_url}}">
+        {{ render_icon('chat-right-quote' )}}
+      </a>
+     </div>
```
Comment (mfisher87) [inline usaon_benefit_tool/templates/base.html:71]:
```suggestion
      <div class="d-flex">
        <a href="{{repo_url}}">
          {{ render_icon('github' )}}
        </a>
        &nbsp;
        &nbsp;
        <a href="{{doc_url}}">
          {{ render_icon('filetype-doc' )}}
        </a>
        &nbsp;
        &nbsp;
        <a href="{{discuss_url}}">
          {{ render_icon('chat-right-quote' )}}
        </a>
      </div>
```

Just an indentation adjustment :)

**Review by mfisher87 — COMMENTED** [review]
Awesome, thanks! Do the suggestions I attached look OK?

File: usaon_benefit_tool/templates/base.html (hunk 1 above)
Comment (mfisher87) [reply usaon_benefit_tool/templates/base.html:55]: Actually, maybe the current year should also be a global variable so it automatically updates!

Comment (Sherwin-14) [reply usaon_benefit_tool/templates/base.html:55]: Should I write the code for the updation of the date periodically?

Comment (mfisher87) [reply usaon_benefit_tool/templates/base.html:55]: If you would like to, that'd be awesome! But I think it's cool to merge this PR if we just change the number to 2024. It feels like we should create a new ticket to auto-update the year. What do you think?

Comment (Sherwin-14) [reply usaon_benefit_tool/templates/base.html:55]: Sure.

**Review by mfisher87 — APPROVED** [review]
Wonderful, thank you again :bow:

**Comment by mfisher87** [comment]
Was just getting back started on this project again after a long time away, and my first thought was how beautiful the footer is :star_struck: 

image

**Pull request merged**